CMakeLists.txt

```diff
@@ -748,9 +747,7 @@ target_link_libraries(ext PRIVATE
 )
 
 if (NOT WIN32)
-  # On Windows we do not link Boost_LIBRARIES here to avoid having multiple defnitions of everything.
-  # We use the header-only version of boost::json: https://www.boost.org/doc/libs/1_87_0/libs/json/doc/html/json/overview.html
-  target_link_libraries(ext PRIVATE pthread ${Boost_LIBRARIES})
```

Comment:
If we are only using the header version of Boost, we don't need to link. Conda-Forge is complaining that it doesn't see us use the boost shared library anywhere, but CMake complains if I only use the Boost headers. Thus, I fixed it here